attest/verifier/config/README.md

```diff
@@ -0,0 +1,80 @@
+attest-verifier-config
+======================
+
+This crate specifies and implements a method for configuring an attestation
+verifier based on sigstructs and other configuration data, which it finds using
+a search path which is given to it. This search path is called the "attestation
+trust root search path".
```

Comment:
:question: This behavior isn't currently here, and I think this PR is big enough on it's own, just wanted to make sure the plan is to add it to this crate

Reply:
whoops, this is stale, thank you